### PR TITLE
14.0 mig auth saml create user

### PR DESCRIPTION
**Pull request by matteoopenf**

### CODE REVIEW

**Comment by matteoopenf** [comment]
Hi @pedrobaeza, I see pre-commit fail in other module instead of the module I migrate
I have that:
************* Module auth_user_case_insensitive.tests.test_res_users
auth_user_case_insensitive/tests/test_res_users.py:51: [E8102(invalid-commit), TestResUsers.test_login_login_is_lowercased] Use of cr.commit() directly - More info https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#never-commit-the-transaction

have you any idea to fix that? or I should open an Issue?

**Comment by matteoopenf** [comment]
@dave@laslabs.com I see give errore is migrate from your company, can you help me?

**Comment by matteoopenf** [comment]
@theangryangel have any idea why fail pre commit?

File: auth_saml_create_user/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "name": "Auth SAML Create User",
+    "summary": """
+        This module extends the functionality of Auth SAML to support
+        the automatic creation of SAML users when they don't exist in odoo.""",
+    "author": "Savoir-faire Linux, Odoo Community Association (OCA)",
+    "maintainers": ["eilst"],
+    "website": "https://github.com/OCA/server-auth",
+    "license": "AGPL-3",
+    "category": "Tools",
+    "version": "14.0.1.0.1",
```
Comment (lorenzoallegrucci) [inline auth_saml_create_user/__manifest__.py]: non deve ripartire da `14.0.1.0.0` ?

**Comment by theangryangel** [comment]
Looking at the actions manifestoo is complaining about the development_status mis-match.

```
auth_saml_create_user (Production/Stable) depends on auth_saml (Beta)
Error: Process completed with exit code 1.
```

Either `auth_saml` needs to be re-classified as `Production/Stable`, or auth_saml_create_user needs to be downgraded to `Beta` to make manifestoo happy :)

I'd suggest that auth_saml_create_user is downgraded personally, but thats just me

**Comment by matteoopenf** [comment]
> Beta

thank you for suggestion

File: auth_saml_create_user/__manifest__.py (hunk 1 above)
Comment (matteoopenf) [reply auth_saml_create_user/__manifest__.py]: https://github.com/OCA/server-auth/pull/517/commits/eb0bdfeb51836e539b3b35039f5d7947cc7614da fix

**Review by lorenzoallegrucci — APPROVED** [review]

**Pull request closed without merging**